snow/networking/handler/health.go

```diff
@@ -54,5 +57,27 @@ func (h *handler) networkHealthCheck() (interface{}, error) {
 		)
 	}
 
+	disconnectedVdrs := h.getDisconnectedValidators()
+	if disconnectedVdrs.Len() > 0 {
+		details["disconnectedValidators"] = disconnectedVdrs
+	}
```

Comment:
Is there a reason that we don't populate this field when everyone is connected? Feels like we can just populate `"disconnectedValidators": h..getDisconnectedValidators()` in the map declaration.

Reply:
thought it would be weird to have it for an empty list.